src/js/other-websites/icon.js

```diff
@@ -19,8 +19,15 @@ async function run() {
   if (!inputIconsAreEnabled) {
     return;
   }
+
+  const pwdMgrCompat = await isPwdMgrCompatEnabled();
+
+  if (pwdMgrCompat) {
+    // TODO: Set RELAY_ICON_WIDTH width, hover state calcs
```

Comment:
This is where the "rest of the work" is for this PR. This PR mostly gets a pref toggled to do the offest, rather than doing it.